src/main/java/Main.java

```diff
@@ -0,0 +1,13 @@
+import core.basesyntax.controller.ConsoleHandler;
+import core.basesyntax.lib.Injector;
+import java.lang.reflect.InvocationTargetException;
```

Comment:
do we need this import?

Reply:
I'm not sure, that I clearly understand you.
import controller.ConsoleHandler -> ConsoleHandler handler = ;
import core.basesyntax.lib.Injector -> Injector.
import java.lang.reflect.InvocationTargetException -> exception in signature main